CONTRIBUTING.md

```diff
@@ -190,6 +190,27 @@ To trace a running worker instance:
 $ ./hack/trace worker
 ```
 
+To attach IDE debugger to a running instance, you can use the `--listen` flag followed by a port and the dlv will be started in headless mode listening on the specified port.   
+
+To debug a running web instance:
+
+```sh
+$ ./hack/trace web --listen 2345
+```
+
+To debug a running worker instance:
+
+```sh
+$ ./hack/trace worker --listen 2345
+```
+
+After this is done, the final step is to connect your IDE to the debugger with the following parameters:
+* host: `localhost`
+* port: `2345` 
+
+For Go Land you can do so by going to Run | Edit Configurations… | + | Go Remote and fill in the parameters.
```

Comment:
Do you mind changing it to `GoLand` without the space? 😁

Reply:
@zoetian My bad. Done.